cpp/src/parquet/encoding.h

```diff
@@ -233,7 +233,7 @@ class DictEncoder : virtual public TypedEncoder<DType> {
 
   /// \brief EXPERIMENTAL: Append dictionary indices into the encoder. It is
   /// assumed (without any boundschecking) that the indices reference
-  /// pre-existing dictionary values
+  /// preexisting dictionary values
```

Comment:
Actually I'm not a native english speaker, so I wonder what's the differences between `pre-existing`  and `preexisting`?

Reply:
Hyphenation is an interim step in English. Eventually some words come into common usage and the hyphenation is dropped.